lib/msal-core/src/cache/AuthCache.ts

```diff
@@ -124,6 +115,20 @@ export class AuthCache extends BrowserStorage {// Singleton
         }
     }
 
+    /**
+     * Reset all temporary cache items
+     */
+    resetTempCacheItems(): void {
+        const storage = window[this.cacheLocation];
+        let key: string;
+        for (key in storage) {
+            // Check if key contains msal prefix; For now, we are clearing all cache items created by MSAL.js
+            if (storage.hasOwnProperty(key) && (Object.keys(TemporaryCacheKeys).indexOf(key) > -1)) {
```

Comment:
I don't believe `storage.hasOwnProperty(key)` check is needed, as it will always be true by nature of doing a for/in.

Reply:
yes. +1